pulsar-broker/src/test/java/org/apache/pulsar/broker/BrokerTestUtil.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker;
+
+/**
+ * Holds util methods used in test.
+ */
+public class BrokerTestUtil {
+    // Generate unique namespace name for different test run.
+    public static String newUniqueNamespace(String prefix) { return prefix + "-" + System.nanoTime(); }
```

Comment:
what about using a static `AtomicLong` variable and use `incrementAndGet` ?

what about naming this method `newUniqueName` ? as we are not using it only for namespaces but also for topic names
or we could have `newNamespaceName` and `newTopicName`

nit: probably formatting is not following current coding style

Reply:
I not sure if just an AtomicLong is enough, if it start from 0 then there's still good chance different test run can have same name, if we initialize with some value like currentTimeMillis() or nanoTime() then I don't see it makes much benefit with the AtomicLong?

Reply:
How about UUID

Reply:
I think that's better, updated to UUID